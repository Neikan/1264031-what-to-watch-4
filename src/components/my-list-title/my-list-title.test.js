// Импорт библиотек
import React from "react";
import renderer from "react-test-renderer";

// Импорт компонентов
import MyListTitle from "./my-list-title.jsx";


describe(`Test MyListTitle component`, () => {
  test(`MyListTitle component is created and rendered correctly`, () => {
    const tree = renderer.create(
        <MyListTitle />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
