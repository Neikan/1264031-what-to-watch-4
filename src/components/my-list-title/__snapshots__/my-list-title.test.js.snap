// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MyListTitle component MyListTitle component is created and rendered correctly 1`] = `
<h1
  className="page-title user-page__title"
>
  My list
</h1>
`;
